packages/testutils/src/serializers/__tests__/form-data-serializer.spec.ts

```diff
@@ -10,11 +10,9 @@ it('serializes FormData with some data', () => {
   const formData = new FormData();
 
   formData.append('name', 'John');
-  formData.append('file', new Blob([], { type: 'image/jpg' }));
 
   expect(formData).toMatchInlineSnapshot(`
     FormData {
-      "file": File {},
```

Comment:
this should work with file serializer

Reply:
Aren't they should be seperated in tests?

Reply:
you should test integration between these tow